### ISSUE TITLE
Snackbar displayed behind system navigation bar

### ISSUE THREAD

**Issue by SimonMarquis** [joreilly/Confetti#1308]
| 3-Button | Gesture |
|--------|--------|
| ![Screenshot_20240425-191702](https://github.com/joreilly/Confetti/assets/1921278/3881c2e4-25fd-4bfb-a8c0-048985823ff4) | ![Screenshot_20240427-115757](https://github.com/joreilly/Confetti/assets/1921278/f3c0228f-08f1-4987-966f-5fbeb4853a53)|

**Comment by waheedkhan-dev** [comment]
@joreilly can i take this issue?

**Comment by SimonMarquis** [comment]
Can't reproduce either. It might have been fixed since I opened this issue.

**Comment by waheedkhan-dev** [comment]
> Can't reproduce either. It might have been fixed since I opened this issue.

Yes, it seems this issue has already been resolved, likely by a recent change or contribution.
